control-plane/.gitignore

```diff
@@ -61,4 +61,10 @@ node_modules/
 # dotenv environment variables file
 .env
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# webpack generated files
+dist/main.css
```

Comment:
How are these files generated? Do we need to add instructions in the README for how to generate them?

Reply:
It will be generated whenever they complete the getting started. If they do yarn deploy, then it auto-generates it for them.

Reply:
Got it. Thanks!